bin/bingx_monitor.py

```
@@ -0,0 +1,226 @@
+            with open(PID_FILE, 'r') as f:
+                pid = int(f.read().strip())
+                
+            # Проверяем, существует ли процесс
+            try:
+                os.kill(pid, 0)
+                return True
+            except OSError:
+                # Процесс не существует, удаляем PID файл
+                os.remove(PID_FILE)
+                return False
+        except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)